routes/documents.js

```diff
@@ -115,23 +115,44 @@ async function renameDocument (req, res, next) {
   const { accessToken } = req
 
   const { siteName, documentName, newDocumentName } = req.params
-  const { sha, content } = req.body
 
   // TO-DO:
   // Validate documentName and content
 
   const { documentDirectory: oldDocumentDirectory, documentFileName: oldDocumentFileName } = extractDirectoryAndFileName(documentName)
   const { documentDirectory: newDocumentDirectory, documentFileName: newDocumentFileName } = extractDirectoryAndFileName(newDocumentName)
 
+  const oldIsomerDocumentFile = new MediaFile(accessToken, siteName)
+  oldIsomerDocumentFile.setFileTypeToDocument(oldDocumentDirectory)
+  const { sha, content } = await oldIsomerDocumentFile.read(oldDocumentFileName)
+  await oldIsomerDocumentFile.delete(oldDocumentFileName, sha)
+
   const newIsomerDocumentFile = new MediaFile(accessToken, siteName)
   newIsomerDocumentFile.setFileTypeToDocument(newDocumentDirectory)
-  const { sha: newSha } = await newIsomerDocumentFile.create(newDocumentFileName, content)
+  await newIsomerDocumentFile.create(newDocumentFileName, content)
+
+  res.status(200).send('OK')
+}
+
+// Move image
```

Comment:
Nit: move document

Reply:
Good catch, fixed in d03948c, additionally fixed the RouteHandler type in the routes